src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -8,6 +8,10 @@ body {
     padding: 60px 0 30px;
 }
 
+a {
```

Comment:
No, all `a` are not by default cursor pointer. By default, they are set to `auto`.